### DISCUSSION TITLE
Fuller Stack Example

### DISCUSSION THREAD

**Discussion by nickreese** [tamagui/tamagui#476] [Ideas]
Hey all, I'll preface that I've been deeply embedded in the Svelte community since React Hooks came and I'm finding the learning curve of putting together all of these libraries pretty steep.

As it currently stands I'm trying to merge the tamagui example with https://github.com/t3-oss/create-t3-turbo and it is quite an undertaking.

I'd love to have an opinionated stack from people who have built several RN+Next apps that includes how data fetching and state management. 

Thank you so much for all of the effort here.

This project is literally seems to be exactly what I needed to get interested in cross platform again.

**Comment by nickreese** [comment]
For anyone in a similar spot, don't miss this: https://github.com/ebg1223/t3-tamagui

Finally got it all wired up and found that haha. At least now I have a deeper understanding of how everything fits together.

**Comment by natew** [comment]
This one was recently released too https://github.com/chen-rn/create-universal-app

**Reply by nickreese** [reply]
Was just studying this last night. Been on the authentication rabbit hole for days and after trying clerk (insane pricing), ory, next-auth (too hacky to get working well on RN at the moment), supertokens, firebase, and supabase... Finally landed on supabase as a solid model for auth. 

Probably means using their SDK instead of prisma which I don't like their types as much as the ones generated by prisma but seems like a worthwhile tradeoff.

**Reply by redbar0n** [reply]
> Been on the authentication rabbit hole for days and after trying clerk (insane pricing), 

I got the attention of Clerk regarding their pricing, and they are open for suggestions to their pricing model if you have any: https://twitter.com/tweetsbycolin/status/1618051497610326017

**Reply by grmkris** [reply]
> Was just studying this last night. Been on the authentication rabbit hole for days and after trying clerk (insane pricing), ory, next-auth (too hacky to get working well on RN at the moment), supertokens, firebase, and supabase... Finally landed on supabase as a solid model for auth.
> 
> Probably means using their SDK instead of prisma which I don't like their types as much as the ones generated by prisma but seems like a worthwhile tradeoff.

@nickreese would be great if you can go down the Prisma path, but still use supabase for auth.. I think we can do it so that migrations + db operations are done through prisma, but auth through supabase. best of both worlds imo
